integration/integration_test.go

```diff
@@ -922,6 +923,7 @@ func (s *IntSuite) TestDisconnectScenarios(c *check.C) {
 				ClientIdleTimeout: services.NewDuration(500 * time.Millisecond),
 			},
 			disconnectTimeout: time.Second,
+			comment:           "1",
```

Comment:
Not sure how to label these test cases but tracking down which one failed only given a dump of the test case struct is not fun.